Wrappers/Python/cil/processors/CentreOfRotationCorrector.py

```diff
@@ -30,36 +30,112 @@ def xcorrelation(slice_index='centre', projection_index=0, ang_tol=0.1):
-        :type FBP: class
-        :param tolerance: The tolerance of the fit in pixels, the default is 1/200 of a pixel. Note this is a stopping critera, not a statement of accuracy of the algorithm.
-        :type tolerance: float, default = 0.001    
-        :param search_range: The range in pixels to search either side of the panel centre. If `None` the width of the panel/4 is used. 
-        :type search_range: int
-        :param initial_binning: The size of the bins for the initial grid. If `None` will bin the image to a step corresponding to <128 pixels. Note the fine search will be on unbinned data.
-        :type initial_binning: int
-        :return: returns an AcquisitionData object with an updated AcquisitionGeometry
-        :rtype: AcquisitionDataS
-        '''
-        processor = CofR_image_sharpness(slice_index=slice_index, FBP=FBP, tolerance=tolerance, search_range=search_range, initial_binning=initial_binning)
+    def image_sharpness(slice_index='centre', backend='astra', tolerance=0.005, search_range=None, initial_binning=None, **kwargs):
+        """
+        This creates a CentreOfRotationCorrector processor which will find the centre by maximising the sharpness of a reconstructed slice.
+
+        Can be used on single slice parallel-beam, and centre slice cone beam geometry. For use only with datasets that can be reconstructed with FBP/FDK.
+
+        Parameters
+        ----------
+
+        slice_index : int, str='centre', default='centre'
+            An integer defining the vertical slice to run the algorithm on.
+
+        backend : str, default='astra'
+            The backend to use for the reconstruction, 'astra' or 'tigre'
+
+        tolerance : float, default=1./200.
+            The tolerance of the fit in pixels, the default is 1/200 of a pixel. This is a stopping criteria, not a statement of accuracy of the algorithm.
+
+        search_range : int, default 0.25*pixels_num_h
+            The range in pixels to search either side of the panel centre. If `None` a quarter of the width of the panel is used.  
+
+        initial_binning : int
+            The size of the bins for the initial grid. If `None` will bin the image to a step corresponding to <128 pixels. Note the fine search will be on unbinned data.
+
+
+        Kwargs
+        ------
+        FBP : Class
+            Deprecated parameter: the FBP class imported from cil.plugins.[backend].FBP Please use 'backend' instead
+
+
+        Example
+        -------
+        from cil.processors import CentreOfRotationCorrector
+
+        processor = CentreOfRotationCorrector.image_sharpness('centre', 'tigre')
+        processor.set_input(data)
+        data_centred = processor.get_ouput()
+
+
+        Example
+        -------
+        from cil.processors import CentreOfRotationCorrector
+
+        processor = CentreOfRotationCorrector.image_sharpness(slice_index=120, 'astra')
+        processor.set_input(data)
+        processor.get_ouput(out=data)
+
+
```

Comment:
These examples don't render. The ones further up in this file work fine.

![image](https://user-images.githubusercontent.com/60604372/180277246-9270e2f5-e4bc-47c0-8dd7-0fd30193f0ed.png)

Reply:
fixed!